proxy/src/main/resources/com/velocitypowered/proxy/l10n/messages_es_ES.properties

```diff
@@ -0,0 +1,61 @@
+#
+# Copyright (C) 2018 Velocity Contributors
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+#
+
+velocity.error.already-connected=¡Ya estás conectado a este servidor!
+velocity.error.already-connected-proxy=¡Ya estás conectado a este proxy!
+velocity.error.already-connecting=¡Ya estás intentando conectarte a un servidor!
+velocity.error.cant-connect=No se ha podido conectar a {0}: {1}
+velocity.error.connecting-server-error=No hemos podido conectarte a {0}. Por favor, inténtalo de nuevo más tarde.
+velocity.error.connected-server-error=Tu conexión a {0} ha sufrido un problema.
+velocity.error.internal-server-connection-error=Se ha producido un error interno en la conexión al servidor.
+velocity.error.logging-in-too-fast=Estás iniciando sesión demasiado rápido, inténtalo de nuevo más tarde.
+velocity.error.online-mode-only=No has iniciado sesión con tu cuenta de Minecraft. Si crees que ya lo estás, intenta reiniciar tu cliente de Minecraft.
+velocity.error.player-connection-error=Se ha producido un error interno en la conexión.
+velocity.error.modern-forwarding-needs-new-client=Este servidor solo es compatible con Minecraft 1.13 y superior.
+velocity.error.modern-forwarding-failed=El servidor no ha mandado una petición de reenvío al proxy. Asegúrate de que tu servidor está configurado para usar el método de reenvío de Velocity.
+velocity.error.moved-to-new-server=Has sido echado de {0}: {1}
+velocity.error.no-available-servers=No hay servidores disponibles a los que conectarte. Inténtalo de nuevo más tarde o contacta con un administrador.
+
+# Commands
+velocity.command.generic-error=Se ha producido un error al ejecutar este comando.
+velocity.command.command-does-not-exist=Este comando no existe.
+
+velocity.command.players-only=Solo los jugadores pueden ejecutar este comando.
+velocity.command.server-does-not-exist=El servidor {0} no existe.
+
+velocity.command.server-current-server=Estás conectado a {0}.
+velocity.command.server-too-many=Hay demasiados servidores registrados. Usa la finalización con tabulación para ver todos los servidores disponibles.
```

Comment:
"la finalización con tabulación" no tiene sentido en el contexto que se da. Para ello se puede usar mejor "el tabulador" o "el autocompletado" para referirse al Tab Complete, de lo contrario, si alguien tiene otra idea para expresarlo mejor, puede sugerirla.

Reply:
As referenced above, that's how [Microsoft calls it in PowerShell](https://docs.microsoft.com/es-es/powershell/scripting/windows-powershell/ise/how-to-use-tab-completion-in-the-script-pane-and-console-pane?view=powershell-7.1), and given the console-like similarities of Minecraft chat commands that's what I found best fitted this message. "Autocompletado" just means "autocompletion" and doesn't tell the user what action (pressing the Tab key) they need to take. "El tabulador" is "the tab" and would translate to "Use the tab to see..." (which doesn't mention pressing tab will autocomplete, the meaning of "seeing" is ambiguous imo). I agree the current message is a bit long, but it's the best I could find in use by a shell.

I did a 5 minute search and that's all I could find. The Spanish Wikipedia article for "Shell" (and others) doesn't mention this feature (unlike the English page 😕). Please share other references with different names and I will change the message. I don't have a Crowdin account, so I couldn't access the translation files for Minecraft, so if someone can share the translated message I will be very thankful.